src/SqlBinding/SqlAsyncCollector.cs

```diff
@@ -268,11 +268,11 @@ public static string GetColumnDefinitionsQuery(string schema, string tableName)
                                 when DATETIME_PRECISION is not null and DATA_TYPE not in ('datetime', 'date', 'smalldatetime') then '(' + cast(DATETIME_PRECISION as varchar(1)) + ')'
 			                    when DATA_TYPE in ('decimal', 'numeric') then '(' + cast(NUMERIC_PRECISION as varchar(9)) + ',' + + cast(NUMERIC_SCALE as varchar(9)) + ')'
 			                    else ''
-		                    end as {ColumnDefinition}	
+		                    end as {ColumnDefinition}
                     from 
 	                    INFORMATION_SCHEMA.COLUMNS c
                     where
-	                    c.TABLE_NAME = '{tableName}'         
+	                    c.TABLE_NAME = {tableName}
```

Comment:
I'd suggest also updating the param names to be `quotedSchema` and `quotedTableName`

Reply:
ping on updating these